newsletter-may-2023.rst

```diff
@@ -0,0 +1,80 @@
+.. post:: May 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - May 2023
+=====================================
+
+News and updates
+----------------
+
+- 🚁️ The proxy application El Proxito that serves all of our documentation websites has a new version that's being slowly rolled out.
+  Several projects are now served using its new URL resolver,
+  which is faster and is receiving new features.
+  We expect to roll it out on all projects soon.
+- 🔎️ ...One of the new features is a better 404 page which has is rolled out on projects that has the new Proxity resolver.
+- 💫️ We now support multiple ``.readthedocs.yaml`` files in the same repository.
+  This is great for monorepos that have several documentation projects with different tooling.
+  Read more about the feature :doc:`in our docs <readthedocs:guides/setup/monorepo>`.
+- ⚙️ If you use ``build.commands`` in ``.readthedocs.yaml``,
+  you are no longer required to have a ``build.tools`` section.
+  We changed the validation for ``.readthedocs.yaml`` to accommodate projects that do not need any of the built-in tools exposed.
+- 🐛️ Fixed: URLs on pull request builds were pointing to the build page,
+  rather than the documentation preview.
+  If a build is successful,
+  the URL now points to the documentation preview.
+- 🔒️ GHSA-4mgr-vrh5-hj8q ...
+
+Upcoming features
+-----------------
+
+- 🚢️ We have concluded work on the version 1.0 of our new `readthedocs-client <https://github.com/readthedocs/readthedocs-client>`_.
+  It's a universal client that enhances documentation for all documentation projects hosted on Read the Docs.
+  We will start to use it on a few selected projects.
+  If you are interested in having it enabled on your project,
+  please reach out.
```

Comment:
This text was a very rough take. I think we should definitely start mentioning the client, but I'm not sure what the framing should be.

@ericholscher if you have a quick idea here, please LMK :+1: The newsletter isn't ready for review yet but I like that we have this on "important" update in place, it gives the newsletter issue some raison d'etre :)

Reply:
I haven't been following the project that closely, but my feeling is that isn't that ready to start promoting it that widely, maybe @agjohnson can have a more accurate opinion.

Reply:
Should "reach out" link to https://readthedocs.org/support/ or similar?

Reply:
Also, I'd sell this a bit more. "It's a universal client that enhances documentation for all documentation projects hosted on Read the Docs." is a fine intro, but it doesn't actually say what it does. Maybe something like:

    The client will be a single library for all the ways Read the Docs enhances documentation from having a flyout menu
    to link to other versions, translations, or subprojects, to showing version warnings for folks browsing outdated docs
    to running ads on community projects.
    There will be integration points and APIs to customize all of these features in the new client.

Reply:
> I haven't been following the project that closely, but my feeling is that isn't that ready to start promoting it that widely

Yeah, this isn't quite that ready yet, it's very beta currently and we are still working out a lot of ops and code patterns still. I'd say this is still alpha level.

> Also, I'd sell this a bit more.

Yeah, agreed here. Related, we are calling these features "addons" instead of "client", particularly for this reason. Though it's not incorrect to call this a client, the function it serves is to enable features in your documentation

Reply:
It sounds like I should remove it then :+1: 

I'll return with the same request for the next newsletter :) I think it's great to build up some hype here. Especially, I think it's a shame that a lot of team resources are going into "invisible" work, meaning that the newsletter can look very quiet when we're not talking about what we are doing.

Reply:
I included the suggestions, @davidfischer, thanks - it's going into the June Newsletter draft :+1: